origins_workflow/origins_workflow.module

```diff
@@ -510,7 +510,7 @@ function origins_workflow_moderation_sidebar_alter(array &$build, EntityInterfac
       $build['actions']['primary']['archive']['#title'] = Markup::create('Archive');
       $build['actions']['primary']['archive']['#type'] = "link";
       // Retrieve alias for current node.
-      $alias = \Drupal::service('path.alias_manager')->getAliasByPath('/node/' . $entity->id());
+      $alias = \Drupal::service('path_alias.manager')->getAliasByPath('/node/' . $entity->id());
```

Comment:
This service name has changed in Drupal 9. Luckily, Drupal 8.9 works with either the old or the new name, so that means that this change fixes Unity and won't break NIDirect (I have tested it)

Reply:
Good catch